content/articles/drop-catching.markdown

```diff
@@ -1,10 +1,26 @@
 ---
-title: Can I use DNSimple for Drop-catching?
-excerpt: Information about DNSimple drop-catching policy.
+title: Can I use DNSimple for Drop catching?
+excerpt: Information about DNSimple drop catching policy.
 categories:
 - DNSimple
+- Domain Registration
+- API
```

Comment:
FYI, each article so far was assigned to exactly one category. It is not a strict requirement (it was in previous systems as of platform requirements) but in order to limit issues I decided to stick with this implicit rule.

Do you feel like it's better to assign multiple categories? As a side note, I don't even know if the platform supports them  as I never tried.